debian/homestead.init.d

```diff
@@ -228,6 +245,17 @@ do_stop()
 }
 
 #
+# Function that runs the daemon/service in the foreground
+#
+do_run()
+{
```

Comment:
We should probably do a `--test` first since calling `run` when the service is started is a bad idea (tm).